pages/MyCrops.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from "react";
+import Grid from "@mui/material/Grid";
+import Button from "@mui/material/Button";
+import ProfileTitle from "../components/ProfileTitle";
+import InputLabel from "@mui/material/InputLabel";
+import MenuItem from "@mui/material/MenuItem";
+import FormControl from "@mui/material/FormControl";
+import Select, { SelectChangeEvent } from "@mui/material/Select";
+import CropsTable from "@/components/CropsTable";
+import Stack from "@mui/material/Stack";
+import { Box } from "@mui/material";
+
+export default function MyCrops() {
+  const [seasonFilter, setSeasonFilter] = React.useState("");
+  const [landFilter, setLandFilter] = React.useState("");
+
+  const handleChange1 = (event: SelectChangeEvent) => {
+    setSeasonFilter(event.target.value);
+  };
+  const handleChange2 = (event: SelectChangeEvent) => {
+    setLandFilter(event.target.value);
+  };
+  return (
+    <Grid container direction="column" rowGap={2}>
+      <Grid item xs={12}>
+        <ProfileTitle title="My Crops" />
+      </Grid>
+
+      <Grid item>
+        <Stack direction="row" spacing={2} paddingTop={2} width={"100%"}>
+          <FormControl variant="filled" sx={{ m: 1, minWidth: 170 }}>
+            <InputLabel id="demo-simple-select-filled-label">
+              Season Filter
+            </InputLabel>
+            <Select
+              labelId="demo-simple-select-filled-label"
+              id="demo-simple-select-filled"
+              value={seasonFilter}
+              onChange={handleChange1}
+            >
+              <MenuItem value="">
+                <em>None</em>
+              </MenuItem>
+              <MenuItem value={10}>Season 1</MenuItem>
```

Comment:
Try to use a loop to generate menu items|
@VishwaLakpriya

Reply:
Fixed